s4e-web/cypress/integration/map/map-locations-search.spec.ts

```diff
@@ -34,23 +34,21 @@ describe.skip('Map Locations Search', () => {
       .resultsShouldBeClosed();
   });
 
-
   it('should clear input', () => {
     LocationsSearch
-      .type('warsz')
-      .waitForCitiesList()
-      .nthResultShouldHaveLabel(0, 'Warszawa')
+      .type('szczeci')
```

Comment:
When you use `searchCitiesBy`, remove `type`

Reply:
I would like make test for adding layer with using modal and button <_Dodaj> how user make this and check that this flow work correctly.